scripts/refresh-diagram-context.sh

```
@@ -87,222 +97,19 @@ if ! ls "$TMP_DIR/diagrams"/*.mmd >/dev/null 2>&1; then
-		name="$(basename "$file" .mmd)"
-		echo "## ${name}"
-		echo
-		echo '```mermaid'
-		cat "$file"
-		echo
-		echo '```'
-		echo
-	done
-} > "$TMP_CONTEXT"
+TMP_CONTEXT_META="$TMP_DIR/diagram-context.meta.json"
+ROOT_DIR="$ROOT_DIR" \
```

Shellcheck (0.11.0)

[warning] This assignment is only seen by the forked process.

(SC2097)